src/components/@shared/atoms/Modal/index.tsx

```diff
@@ -1,8 +1,9 @@
 import React, { ReactElement, ReactNode } from 'react'
 import ReactModal from 'react-modal'
-import styles from './Modal.module.css'
+import styles from './index.module.css'
 
-if (process.env.NODE_ENV !== 'test') ReactModal.setAppElement('#__next')
+if (process.env.NODE_ENV !== 'development' && process.env.NODE_ENV !== 'test')
```

Comment:
which tool sets `development` to justify this change? Otherwise no reason to touch this line at all